agreement/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Agreement',
+    'summary': "Adds an agreement object",
+    'version': '12.0.1.0.0',
+    'category': 'Contract',
+    'author': "Akretion,Odoo Community Association (OCA)",
+    'contributors': 'Yves Goldberg (Ygol InternetWork)',
```

Comment:
I am not sure this key exists.

I would add the `development_status` and the `maintainers` one though.

Reply:
Should I remove 'contributors' and add myself as author?
Which 'development_status' ?
Who is the the 'maintainers' ?
tnx